.github/workflows/dotnet.yml

```diff
@@ -25,15 +27,18 @@ jobs:
     - name: Build (TestFlashLFQ)
       run: cd mzLib && dotnet build --no-restore ./TestFlashLFQ/TestFlashLFQ.csproj
     - name: Add coverlet collector (Test)
```

Comment:
Can these be removed because you added the coverlet PackageReference to these csproj files?

Reply:
I think you're right. However, the changes I made in this pull request were unnecessary and broke test coverage in FlashLFQ.